### PR TITLE
Fix warnings

### LINKED ISSUE #424: Apply all flutter analyzer fixes
**Is your feature request related to a problem? Please describe.**
There are currently many issues which can be fixed automatically or semi-automatically.

### PR DESCRIPTION
**Pull request by maxammann**
fixes #424

### CODE REVIEW

File: frontend/lib/store_widgets/detail/detail_content.dart [hunk 1]
```diff
@@ -53,7 +53,6 @@ class DetailContent extends StatelessWidget {
               ],
               Column(
                 children: <Widget>[
-                  if (address != null)
```
Comment (klinzo) [inline frontend/lib/store_widgets/detail/detail_content.dart:56]: Indentation/linting? Is there an issue for this?

**Review by klinzo — APPROVED** [review]
Would it be possible to change these things from warnings to errors such that it won't be forgotten in the future?

File: frontend/lib/store_widgets/detail/detail_content.dart (hunk 1 above)
Comment (maxammann) [reply frontend/lib/store_widgets/detail/detail_content.dart:56]: https://www.elitenetzwerk.de/portal/user/home/main/query?eventId=2114

Comment (klinzo) [reply frontend/lib/store_widgets/detail/detail_content.dart:56]: What is this?

Comment (maxammann) [reply frontend/lib/store_widgets/detail/detail_content.dart:56]: ups wrong thing in clipboard :D

Comment (maxammann) [reply frontend/lib/store_widgets/detail/detail_content.dart]: I added a circleci job in this PR for validating that it is formatted

**Pull request merged**